app/src/test/kotlin/compiler/semantic_analysis/VariablePropertiesAnalyzerTest.kt

```diff
@@ -440,4 +477,79 @@ class VariablePropertiesAnalyzerTest {
         assertAnalysisResults(input, expectedResults)
         assertDiagnostics(input, listOf())
     }
+
+    // czynność zewnętrzna() {
+    //     czynność wewnętrzna(x: Liczba = 123) { }
+    //     wewnętrzna(124)
+    // }
+
+    @Test
+    fun `test uncalled default parameter`() {
+        val parameterX = Function.Parameter("x", Type.Number, Expression.NumberLiteral(123))
+        val dummyVariableX = Variable(Variable.Kind.VALUE, "dummy", Type.Number, Expression.NumberLiteral(123))
+        val innerFunction = Function("wewnętrzna", listOf(parameterX), Type.Unit, listOf())
+        val innerFunctionCall = Expression.FunctionCall("wewnętrzna", listOf(Expression.FunctionCall.Argument("x", Expression.NumberLiteral(124))))
+        val function = Function(
+            "zewnętrzna", listOf(), Type.Unit,
+            listOf(
+                Statement.FunctionDefinition(innerFunction),
+                Statement.Evaluation(innerFunctionCall),
+            )
+        )
+        val nameResolution: MutableReferenceMap<Any, NamedNode> = ReferenceHashMap()
+        nameResolution[innerFunctionCall] = innerFunction
```

Comment:
```suggestion
        val nameResolution: MutableReferenceMap<Any, NamedNode> = referenceMapOf(innerFunctionCall to innerFunction)
```
Maybe we should unify all similar calls to this convention as well